QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -487,9 +410,7 @@ def qualcomm_efficient_converter(
             tokenizer=tokenizer,
             onnx_dir_path=onnx_dir_path,
             seq_length=seq_length,
-            return_path=return_path,
-            save_fp16_onnx=save_fp16_onnx,
-            save_fp32_onnx=save_fp32_onnx)
+            return_path=return_path)
```

Comment:
Is this required now? We can always return path of saved `clipped_fp32` model onnx file?